_posts/2024-03-03.quarkus-redhat-strategy.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: 'Quarkus & Red Hat''s evolving middleware strategy'
+date: 2025-03-03
+tags: announcement
+synopsis: 'Information about the recent announcement and the future of Quarkus'
```

Comment:
And its influence on the cure of Quarkus

Reply:
you mean like this ?

```suggestion
synopsis: 'Information about the recent announcement and its influence on Quarkus'
```

Reply:
yes